rellic/AST/Z3ConvVisitor.cpp

```diff
@@ -790,13 +847,36 @@ void Z3ConvVisitor::VisitBinaryApp(z3::expr z_op) {
       }
     } break;
 
+    case Z3_OP_XOR:
+      c_op = CreateBinaryOperator(*ast_ctx, clang::BO_Xor, lhs, rhs,
+                                  ast_ctx->BoolTy);
+      break;
+
     case Z3_OP_BADD:
       c_op = CreateBinaryOperator(*ast_ctx, clang::BO_Add, lhs, rhs,
                                   GetIntResultType());
       break;
 
+    case Z3_OP_BSUB:
+      c_op = CreateBinaryOperator(*ast_ctx, clang::BO_Sub, lhs, rhs,
+                                  GetIntResultType());
+      break;
+
+    case Z3_OP_BMUL:
+      c_op = CreateBinaryOperator(*ast_ctx, clang::BO_Mul, lhs, rhs,
+                                  GetIntResultType());
+      break;
+
+    case Z3_OP_BSDIV_I:
+    case Z3_OP_BUDIV_I:
+      c_op = CreateBinaryOperator(*ast_ctx, clang::BO_Div, lhs, rhs,
```

Comment:
Do I want non-`_I` variants here, too? I've been avoiding floats in all my PRs

Reply:
Yeah, I think we can omit floats until someone explicitly needs or wants them in.